config/index.js

```diff
@@ -472,6 +474,25 @@ var conf = convict({
       default: true,
       env: 'SECURITY_HISTORY_ENABLED'
     }
+  },
+  lastAccessTimeUpdates: {
+    enabled: {
+      doc: 'enable updates to the lastAccessTime session token property',
+      format: Boolean,
+      default: false,
+      env: 'LASTACCESSTIME_UPDATES_ENABLED'
+    },
+    sampleRate: {
+      doc: 'sample rate for updates to the lastAccessTime session token property, in the range 0..1',
+      format: Number,
+      default: 1,
+      env: 'LASTACCESSTIME_UPDATES_SAMPLE_RATE'
+    },
+    enabledEmailAddresses: {
+      doc: 'regex matching enabled email addresses for updates to the lastAccessTime session token property',
+      default: '.+@mozilla\\.com$',
```

Comment:
Fwiw, I've opened a convict PR to add `RegExp` support at https://github.com/mozilla/node-convict/pull/165. I'm not supremely confident about it being accepted, so let's see how it goes.